### DISCUSSION TITLE
Translations for Asian letters

### DISCUSSION THREAD

**Discussion by Madis0** [C1710/blobmoji#73] [Ideas]
Since most emoji that have text are in English, I've got an idea: why not add "subtitles" for Japanese/Chinese letter emoji too? 

The "subtitles" could consist of one or two words/abbrevations and meanings can be taken from Emojipedia. 

Optionally, you could make some English-only emoji by for example combining 🈶🇬🇧 but since that would work only here anyway, I wouldn't consider a priority.

**Comment by C1710** [comment]
This is an interesting idea, but I'm not sure if this works. Even an abbreviation could be too long for one character width. I might take a look at it when I have time for this again 😅

**Comment by Madis0** [comment]
My ideas:

| Emoji | Caption | Emojipedia translation |
|-------|---------|------------------------|
| 🈁     | HERE    | "Here"                 |
| 🈂️     | GIVN    | "Service charge"       |
| 🈷️     | MO.    | "Monthly Amount"       |
| 🈶     | PAID    | "Not free of charge"   |
| 🈯     | RSVD    | "Reserved"             |
| 🉐     | BRGN    | "Bargain"              |
| 🈹     | SALE    | "Discount"             |
| 🈚     | FREE    | "Free of charge"       |
| 🈲     | DON'T   | "Prohibited"           |
| 🉑     | OK      | "Acceptable"           |
| 🈸     | APLY    | "Application"          |
| 🈴     | AGRE    | "Passing grade"        |
| 🈳     | EMPT    | "Vacancy"              |
| ㊗️     | GRTZ    | "Congratulations"      |
| ㊙️     | SCRT    | "Secret"               |
| 🈺     | WORK    | "Open for business"    |
| 🈵     | FULL    | "No vacancy"           |

**Comment by C1710** [comment]
My idea for the mid/long term would be to just replace the Japanese letters in an extension pack.
